### PR TITLE
[plearn] Add plearn strategy 

### PR DESCRIPTION
**Pull request by jeeraponnnnnnnn**
Changes proposed in this pull request:
- Add a voting strategy for the Plearn project, which calculates voting power based on holdings of the project(pools, token).

### CODE REVIEW

File: src/strategies/plearn/index.ts [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        addresses.map((address: any) => [
+          item.address,
+          'userInfo',
+          [address],
+          { blockTag }
+        ]),
+        { blockTag }
+      )
+    )
+  );
```
Comment (ChaituVR) [inline src/strategies/plearn/index.ts:139]: Is it possible to limit the number of `lockedPoolAddresses` `foundingInvestorPoolAddresses`, `pendingWithdrawalAddresses` and make sure number of requests doesn't cross 5 https://github.com/snapshot-labs/snapshot-strategies#code

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Also some of test cases are failing, can take a look at them plz? :)

File: src/strategies/plearn/index.ts (hunk 1 above)
Comment (ChaituVR) [reply src/strategies/plearn/index.ts:53]: @jeeraponnnnnnnn

Comment (jeeraponnnnnnnn) [reply src/strategies/plearn/index.ts]: @ChaituVR 
I've implemented the changes as you suggested. Now, I've limited the arrays for lockedPoolAddresses, foundingInvestorPoolAddresses, and pendingWithdrawalAddresses to have no more than 5 addresses each. Thanks

Comment (ChaituVR) [reply src/strategies/plearn/index.ts]: But that will max 15 network calls, we can only allow upto max 5 network calls https://github.com/snapshot-labs/snapshot-strategies#code

Comment (jeeraponnnnnnnn) [reply src/strategies/plearn/index.ts]: @ChaituVR  I've updated code, could you please take another look at it?

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
@jeeraponnnnnnnn could you resolve conflicts please :)

**Pull request merged**